packages/expo-cli/src/commands/start/TerminalUI.js

```diff
@@ -202,7 +202,6 @@ export const startAsync = async (projectDir, options) => {
               try {
                 await Exp.sendAsync(sendTo, lanAddress);
                 log(`Sent link successfully.`);
-                sent = true;
```

Comment:
i think we should actually keep this since we use `sent` below in https://github.com/expo/expo-cli/pull/1362/files#diff-dfafc98a4c32ef6db0217cf6b6fcc7c7R210

Reply:
OK cool.